pkg/reconciler/v1alpha1/route/route.go

```diff
@@ -52,6 +55,11 @@ const (
 	controllerAgentName = "route-controller"
 )
 
+var (
+	routeResource  = v1alpha1.Resource("routes")
+	routeFinalizer = routeResource.String()
```

Comment:
Add some doc comments for these semi-const?

Reply:
Done